DUI3-DX/Converters/ArcGIS/Speckle.Converters.ArcGIS3/Utils/ArcGISProjectUtils.cs

```diff
@@ -0,0 +1,66 @@
+using ArcGIS.Core.Data;
+using ArcGIS.Core.Data.DDL;
+using ArcGIS.Desktop.Core;
+using ArcGIS.Desktop.Framework.Threading.Tasks;
+
+namespace Speckle.Converters.ArcGIS3.Utils;
+
+public class ArcGISProjectUtils
```

Comment:
yeah let's interace this out and inject it plz

Reply:
will need some help here